lib/sqlalchemy/dialects/postgresql/psycopg.py

```diff
@@ -117,6 +117,20 @@ def close(self):
         creator=mypool.getconn,       # Use Psycopg 3 connection pool to obtain connections
     )
 
+Usage of ``psycopg_pool.AsyncConnectionPool`` is similar, but it requires the
+following ``psycopg.AsyncConnection`` subclass instead:
```

Comment:
I think it make sense to mention also the use of the AsyncConnectionPool and async_creator

Reply:
@CaselIT - good feedback. There were indeed enough differences that a more complete example was needed. It's there now.